Antidote/CallsManager.m

```diff
@@ -271,6 +275,43 @@ - (void)ringingCallDeclineButtonPressed:(RingingCallViewController *)controller
                             error:nil];
 }
 
+#pragma mark - ActiveCallViewControllerDataSource
+
+- (NSInteger)activeCallControllerNumberOfPausedCalls:(ActiveCallViewController *)controller
+{
+    return [[self.allPausedCallsController fetchedObjects] count];
+}
+
+- (NSString *)activeCallController:(ActiveCallViewController *)controller pausedCallerNicknameForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    OCTFriend *friend = [call.chat.friends firstObject];
```

Comment:
Oh, sorry, if you started the call then `caller` will be nil. Then everything is correct.